service/dap/config.go

```diff
@@ -85,3 +94,37 @@ func configureSetSubstitutePath(args *launchAttachArgs, rest string) error {
 	}
 	return nil
 }
+
+func configureSetShowPprofLabels(args *launchAttachArgs, rest string) error {
```

Comment:
Could you add documentation to command.go explaining how this configuration works please? For msgConfig

Reply:
Added in 285116f8256e